docs/proposal/3_5_schedule.tex

```diff
@@ -1,4 +1,10 @@
 \subsubsection{Major Schedule}
+
+FIXME: we should add a timeline chart with labels of milestone I, II, and III
```

Comment:
I don't know if a literal chart is necessary, but I agree that a timeline is important.